Flick/Controllers/ProfileViewController.swift

```diff
@@ -128,10 +128,10 @@ class ProfileViewController: UIViewController {
     }
 
     private func getCurrentUser() {
-        NetworkManager.getUserProfile { [weak self] userProfile in
-            guard let self = self else { return }
+        NetworkManager.getUserProfile { [weak self] userProfile, success in
+            guard let self = self, success else { return }
             DispatchQueue.main.async {
-                self.updateUserInfo(user: userProfile)
+                self.updateUserInfo(user: userProfile!)
```

Comment:
i think we shouldn't force unwrap it right. it would crash if request fails and we got nil for userProfile. 
we can do add `if let` or `guard let`